app/src/main/java/com/jjswigut/matters/ui/MatterListRecyclerViewAdapter.kt

```diff
@@ -31,12 +28,85 @@ class MatterListRecyclerViewAdapter(
 
     override fun getItemCount(): Int = values.size
 
+    fun submitList(dummyList: List<DummyMatter>) {
+        values = dummyList
```

Comment:
in general we want to update our data & adapter with something called diff util (https://developer.android.com/reference/androidx/recyclerview/widget/DiffUtil) 

it's a way of intelligently updating the recyclerview and only updating the items that actually changed